README.md

```diff
@@ -1,5 +1,10 @@
 [![Build Status](https://travis-ci.org/grpc/grpc-swift.svg?branch=master)](https://travis-ci.org/grpc/grpc-swift)
 
+# ‼️ Deprecation Notice ‼️
+
+`v0.x` **will soon be deprecated in favor of a new implementation, please see
```

Comment:
nit: replace `,` with `;`

Also, I would provide a few more info here. Maybe "gRPC Swift versions 0.x based on gRPC-Core will soon be replaced with a re-implementation based on SwiftNIO. We strongly suggest using the [`nio` branch](https://github.com/grpc/grpc-swift/tree/nio) for new projects. Please refer to [DEPRECATION.md](DEPRECATION.md) for more information on the deprecation process."